kaldb/src/main/java/com/slack/kaldb/server/Kaldb.java

```diff
@@ -72,11 +72,14 @@ public void setup() {
     serviceManager.startAsync();
   }
 
-  public static Set<Service> getServices() {
+  public static Set<Service> getServices() throws Exception {
```

Comment:
This isn't needed, no exception is currently thrown.

Reply:
Fixed.